wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+# Glossaire
+
+Pendant des années, la osu!community a adopté son propre verlan qui peut être déroutant pour les débutants.
+Cette page est une compilation de la plupart de ces termes.
+Les ajouts et modifications sont les bienvenus !
+
+<!--Les liens préfixés par le symbole "#" sont aussi des termes de ce glossaire, tandis que les liens bleus pointent vers d'autres articles.-->
+
+## A
+
+### Friche abandonnée (terme original : Abandoned wasteland)
+
+Pour ceux qui trouvent que le nom n'est pas assez évident, la friche abandonnée est un sous-forum caché ou les instructions données aux modérateurs en charge sont "Mettez dedans les choses dont nous n'avons pas besoin".
+Quand un thread est considéré comme "wastelanded", il est déplaçé dans ce sous-forum.
+
+### AFK
+
+AFK est un acronyme anglais qui signifie "Away From Keyboard" (littéralement : "loin du clavier").
+Par exemple quand un utilisateur part faire quelque chose d'autre, alors qu'il est encore connecté.
+osu! désigne les joueurs inactifs depuis un certain temps avec la mention "Idle".
+
+### Cercle d'approche
+
+Dans le mode de jeu [osu!standard](/wiki/Game_Modes/osu!/), le cercle d'approche est le cercle extérieur qui entoure le cercle de frappe, le debut d'un slider, ou le centre d'un spinner.
+Quand le cercle d'approche touche la bordure extèrieure du cercle de frappe ou du spinner, vous devez cliquer.
+Pour les spinners, le cercle d'approche indiquera la durée de ce spinner.
+
+### Authentique (terme original : Authentic)
+
+"Authentic" est un terme qui désigne une beatmap qui reprend tous les aspects d'un niveau spécifique des jeux DS originaux (chanson, paramètres, placement des objets, etc.).
+
+## B
+
+### Bancho
+
+Bancho est le serveur "aisément expensible" codé par peppy.
+Il supplante le besoin d'un serveur IRC tiers pour le management du contenu online d'osu!.
+Les joueurs se connectent automatiquement à Bancho quand ils démarrent osu!, et ce si ils sont connectés à internet
+
+### BAT
+
+**Au 1° février 2015, le BAT a été renommé le Beatmap Nomination Group**
+
+BAT&mdash;acronyme pour Beatmap Appreciation Team&mdash;était un groupe d'utilisateurs qui se chargeaient d'approuver et de ranker vos beatmaps .
+
+### Pulsation (terme original : Beat)
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Pulsation

_Terme original : Beat_
```